src/containers/Tenant/Query/QuerySettingsBanner/QuerySettingsBanner.tsx

```diff
@@ -0,0 +1,32 @@
+import {Alert} from '@gravity-ui/uikit';
+
+import {cn} from '../../../../utils/cn';
+import {useChangedQuerySettings} from '../../../../utils/hooks/useChangedQuerySettings';
+import i18n from '../i18n';
+
+const b = cn('ydb-query-settings-banner');
+
+import './QuerySettingsBanner.scss';
+
+export function QuerySettingsBanner() {
```

Comment:
Banner isn't properly aligned - text is not in the middle, close button not on the same level with text

<img width="1215" alt="Screenshot 2024-07-31 at 12 24 54" src="https://github.com/user-attachments/assets/82906f68-fca4-4c12-a4fb-677a6137b381">

Reply:
fixed with align prop in Alert